src/app/Support/Support.tsx

```diff
@@ -1,40 +1,74 @@
 import * as React from 'react';
-import { CubesIcon } from '@patternfly/react-icons';
-import {
-  PageSection,
-  Title,
-  Button,
-  EmptyState,
-  EmptyStateVariant,
-  EmptyStateIcon,
-  EmptyStateBody,
-  EmptyStateSecondaryActions
-} from '@patternfly/react-core';
+import {Button, Modal, Stack, StackItem, Text, TextContent, TextVariants} from '@patternfly/react-core';
+import {global_FontSize_4xl, global_spacer_sm, global_spacer_md, global_warning_color_100} from "@patternfly/react-tokens";
+import {ExclamationTriangleIcon} from "@patternfly/react-icons";
+
+/**
+ * Support pop up when the user has run the server and there is no user/password to log in
```

Comment:
@karesti there's always a user/password combination now, the server generates one and logs it in the console:

```
# IDENTITIES_PATH not specified                                                #
# USER and/or PASS env variables not specified.                                #
# Auto generating user and password.                                           #
```

If I cancel the HTTP authentication dialog twice, I don't get this page, I get an empty console instead.
The only message in the browser's console is "Unable to retrieve server version About.tsx:40:16"

Reply:
yep, this is happening when you run a container. running the server without the container is different, you need to create a user/pass and the pop up appears.
let me try what you mention here

Reply:
give a try dan please